packages/whenable/README.md

```diff
@@ -41,7 +43,7 @@ const b = await E(w).something(...args);
 Use the following to create and resolve a whenable:
 
 ```js
-import { makeWhenableKit } from '@agoric/whenable/heap.js';
+import { makeWhenableKit } from '@agoric/internal/whenable.js';
 [...]
 const { settler, whenable } = makeWhenableKit();
```

Comment:
May be silly, but I'd always write

```suggestion
const { whenable, settler } = makeWhenableKit();
```

because when I use a promiseKit, I always mention the promise before the resolver.

Reply:
I still prefer the semantics-based order. But happy to resolve this one with no change requested.